app/graphql/mutations/export_to_csv.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Mutations
+  class ExportToCsv < BaseMutation
+    field :search_export, Types::SearchExportType, null: true
+
+    argument :search_hash, String, required: true
+    argument :site_view_id, Integer, required: true
+
+    def resolve(site_view_id:, search_hash:)
```

Comment:
If we were using [global object identification](https://github.com/rmosolgo/graphql-ruby/blob/master/guides/relay/object_identification.md) we would be able to retrieve the site view and set `site_view` as the param.